stable/spring-cloud-data-flow/templates/server-config.yaml

```diff
@@ -23,6 +23,9 @@ data:
                     limits:
                       memory: {{ .Values.deployer.resourceLimits.memory }}
                       cpu: {{ .Values.deployer.resourceLimits.cpu }}
+{{- if .Values.server.containerConfiguration }}
```

Comment:
Probably a dumb question. Any reason why this is indented differently?

Reply:
the `indent 10` below formats the toYaml block